update docs (#347)
tier4/scenario_simulator_v2: docs/tutorials/RunWithDocker.md

If you want to run scenario_simulator_v2 with docker, please follow the instructions below.

> Currently, mac OS and Windows does not support yet.

Reviewer: nits: 
```suggestion
Currently, macOS and Windows are not supported yet.
```

Author: Thank you for helping my poor English!
